packages/flutter/lib/src/foundation/diagnostics.dart

```diff
@@ -178,6 +235,13 @@ class TextTreeConfiguration {
   /// Prefix to add to the first line to display a child with this style.
   final String prefixLineOne;
 
+  /// Similar to prefixLineOne but applies even when this is the root node in
```

Comment:
It's unclear for me what this does based on the description.

Reply:
Updated the comment and moved this next to the related afterName property.